external-docs/docs/index.md

```diff
@@ -1,17 +1,23 @@
 # Welcome to MkDocs
 
-For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+## The Prject
+### What the Project is about
 
-## Commands
+The csvcubed project aims to provide libraries and utilities which make it simple to turn a CSV into 5* linked data. We want to make it easier for data producers to annotate their existing data with the metadata required to ensure that each dataset is discoverable, comparable and analysable by automated tools.
```

Comment:
Could you make sure to include the links that were in the dev docs too? I know `5* linked data` used to be a link.